modules/con_quay_openshift_faq_op_v2.adoc

```diff
@@ -0,0 +1,12 @@
+[[con-quay-openshift-faq-op-v2]] 
+
+= Frequently Asked Questions for the {productname} Operator
+
+== What is a managed component?
+
+== What is the PostgreSQL managed component?
+
+== What is the Storage managed component?
+
+== What is the Redis managed component?
+
```

Comment:
What are the upgrade steps from v1 to v2?
Can a managed component be released to be self-managed?
How are certs handled?
What monitoring is available?